src/issue/screens/issue.screen.js

```diff
@@ -293,11 +324,14 @@ class Issue extends Component {
       isPendingIssue ||
       isDeletingComment
     );
-    const isShowLoadingContainer = isPendingComments || isPendingIssue;
-    const fullComments = !isPendingComments ? [issue, ...comments] : [];
+    const isShowLoadingContainer =
+      isPendingComments || isPendingIssue || isPendingEvents;
+    const fullEvents = !isPendingComments
```

Comment:
Since `fullEvents` is essentially everything (issue description, comments and events), maybe we can find a better name for this. `fullConversation` or something? (Not sure if that's any better however :joy: :sob:)

Reply:
agreed! i hesitated when i wrote that name but couldn't come up with anything better. `fullConversation` seems like the right move. i don't see it used in `git-point` yet, but it's used on github:

![screen shot 2017-10-08 at 3 58 44 pm](https://user-images.githubusercontent.com/820696/31320386-b635f7ea-ac41-11e7-8f2c-87b346fe2b0f.png)

Reply:
i [took care of this naming](https://github.com/gitpoint/git-point/pull/438/commits/0bb54159e3abe39277524de9a004646f58b11f6c), while extracting that chunk of logic into `utils/` for future testing.